src/avram/tasks/db/migrations_status.cr

```diff
@@ -0,0 +1,42 @@
+require "shell-table"
+
+class Db::Migrations::Status < LuckyCli::Task
+  summary "Print the current status of migrations"
+
+  def call
+    if migrations.none?
```

Comment:
I love the use of `none?` 👍

Reply:
Crystal has been awesome to work in so far, I love little methods like this